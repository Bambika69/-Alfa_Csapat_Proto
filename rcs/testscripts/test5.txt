createfield -t laboratory
createvirologist L1
startturn V1
search
learngeneticcode