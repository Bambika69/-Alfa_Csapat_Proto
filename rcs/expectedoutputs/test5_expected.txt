Field created:
ID: L1
Type: Laboratory
Neighbours: null

Virologist created:
ID: V1
Field: L1
Backpack: B1

  V1â€™s turn started.

  Virologist searched:
Field: L1
Type: Laboratory
Neighbours: null
Virologists: V1
GeneticCodes: null
Equipments: null
Elements: null

Genetic code learned:
Virologist: V1
Laboratory: L1
GeneticCode: null
Result: failed
